microsoft-edge/extensions-chromium/store-policies/developer-policies.md

```diff
@@ -343,8 +343,8 @@ Users should not be misled to click on advertisements to fully use the extension
 ###### 1.10.2 Policies and agreements
 
 Any ad content that's displayed by your extension must conform to the following:
-* The [Microsoft Advertising Network policies](https://about.ads.microsoft.com/policies/home).
-* The advertising requirements of the [App Developer Agreement](/legal/windows/agreements/app-developer-agreement).
+* The [Microsoft Advertising Network policies](https://about.ads.microsoft.com/policies/home).<!-- todo: update link text & url - the link redirects to [Introduction to Microsoft Advertising Network policies](https://help.ads.microsoft.com/#apex/ads/en/ext60205) -->
```

Comment:
update link text & url - the link redirects to [Introduction to Microsoft Advertising Network policies](https://help.ads.microsoft.com/#apex/ads/en/ext60205) -->
```suggestion
* The [Microsoft Advertising Network policies](https://help.ads.microsoft.com/#apex/ads/en/ext60205).
```
live article has:
[Microsoft Advertising Network policies] https://about.ads.microsoft.com/policies/home
proposed change:
[Microsoft Advertising Network policies] https://help.ads.microsoft.com/#apex/ads/en/ext60205
though actual title of dest page is "Introduction to ..." and it's not clear to me whether we mean one webpage of a set of ten webpages.
@Bhuvana-Priya @sivMSFT

Reply:
the latest correct webpage to link to appears to be
"Microsoft Advertising policies and agreements"
https://help.ads.microsoft.com/#apex/ads/en/n5095/0
![image](https://github.com/user-attachments/assets/64446154-2e5d-442b-97bf-d0cd74eee141)

i'll push a commit to use that link text and url

Reply:
decided to go with the top-level page and its displayed title (rather than the Intro page that the url is redir'd to, and its 3 different titles it mentions, one of which matches our orig link text).  
result:
```
* [Microsoft Advertising policies and agreements](https://help.ads.microsoft.com/#apex/ads/en/n5095/0).
```